solver/src/oneinch_solver/api.rs

```diff
@@ -66,6 +66,8 @@ pub struct SwapQuery {
     pub from_address: H160,
     /// Limit of price slippage you are willing to accept.
     pub slippage: Slippage,
+    /// Flag to disable checks of the required quantities.
+    pub disable_estimate: Option<bool>,
```

Comment:
If None is the same as Some(false) maybe we have the raw bool instead of Option. Not sure what is better.

Reply:
The query accepts `disableEstimate=false` as well. In theory, its an optional flag with a default value which is `false` ATM, but we may want to force it to `false` if that were to change. So semantically there are 3 options:
- `None`: use default
- `Some(false)`: force enable estimate
- `Some(true)`: force disable estimate

Also, it keeps it more 1:1 with the API.